models/classes/ResultService.php

```diff
@@ -29,6 +29,12 @@
 
     const CONFIG_ID = 'qtiResultsService';
 
+    const PROPERTY_DELIVERY_RESULT	='http://www.tao.lu/Ontologies/TAOResult.rdf#DeliveryResult';
+
+    const PROPERTY_SUBJECT	='http://www.tao.lu/Ontologies/TAOResult.rdf#resultOfSubject';
```

Comment:
These are not properties, juste simple uris

Reply:
Renamed to CLASS_URI as suflix